Extension/src/main.ts

```diff
@@ -428,7 +424,7 @@ async function postInstall(info: PlatformInformation): Promise<void> {
 
     // If there is a download failure, we shouldn't continue activating the extension in some broken state.
     if (!installSuccess) {
-        return Promise.reject<void>("");
+        throw new Error(``);
```

Comment:
Hmm.. Maybe we should use a meaningful error message here.

Reply:
I added a message here. does handleError take care of sending the good error message in telemetry?